definitions/procedures/restore/regenerate_queues.rb

```diff
@@ -0,0 +1,63 @@
+module Procedures::Restore
+  class RegenerateQueues < ForemanMaintain::Procedure
+    metadata do
+      description 'Regenerate required qpidd and activemq queues while restoring online backup'
+    end
+
+    def ssl_cert
+      "/etc/pki/katello/certs/#{hostname}-qpid-broker.crt"
+    end
+
+    def ssl_key
+      "/etc/pki/katello/private/#{hostname}-qpid-broker.key"
+    end
+
+    def amqps_url
+      'amqps://localhost:5671'
```

Comment:
ditto

Reply:
+1

Reply:
I see the port is stored in the answers file, this is probably the thing I would worry most about changing in the future:

```
/etc/foreman-installer/scenarios.d/satellite-answers.yaml:  qpid_router_broker_port: 5671
```

We do access the answers file for other purposes https://github.com/theforeman/foreman_maintain/blob/7e1ac6d6edc0e5ad7107d7a626cefafda7b2faba/definitions/features/katello.rb#L64-L66

Perhaps you could do the same here for the port?

Reply:
Yes, this is good suggestion. However I have never seen anyone who changes the default port of qpidd on Satellite. Or maybe I can even say I have not seen someone chaning the Satellite services ports.

Reply:
> However I have never seen anyone who changes the default port of qpidd on Satellite. Or maybe I can even say I have not seen someone chaning the Satellite services ports

It keeps things flexible so that we don't have to update a ton of repos if we ever want to change that port or for some reason a customer needs to use a different port. We tend to try to use the answers file as a single source of truth